modules/mobile/spec/support/helpers/spec_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+RSpec.configure do |config|
+  config.before :each, type: :request do
+    Flipper.enable('va_online_scheduling')
```

Comment:
The fact that we were doing this in the iam helper is interesting. That suggests that this has been buggy for years and that we were previously jamming it into the iam helper to fix the issue. Very mysterious. 

Ideally, we wouldn't be necessary because it doesn't apply to most endpoints and should just be on by default. Since we're essentially doing this to patch a mysterious flipper bug, I think we should leave a comment explaining the situation.

Reply:
Sure. Comment added.